connector/queue/model.py

```diff
@@ -81,6 +81,7 @@ class QueueJob(models.Model):
     eta = fields.Datetime(string='Execute only after')
     active = fields.Boolean(default=True)
     model_name = fields.Char(string='Model', readonly=True)
+    record_ids = fields.Char(string='Record IDs', readonly=True)
```

Comment:
They use a JSON serialized field in later versions upstream: https://github.com/OCA/queue/blob/13.0/queue_job/models/queue_job.py#L41. Do we need to take extra care that we store the record_ids as valid json? Can we use that for more efficient indexing? Maybe there are other opportunities to align to upstream code (but maybe the code was refactored beyond recognition).